### ISSUE TITLE
scraper ötlet: kormany.hu/dokumentumtar

### ISSUE THREAD

**Issue by pisipite** [law909/bmm-protoscraper#1]
a kormany.hu dokumentumtárába kerülnek fel a minisztériumok közleményei, ezekre hasznos lenne egy új scrapert építeni.
a dokumentumok jellege többek között:
- társadalmi egyeztetésekről
- jogszabálytervezetekről
- személyzeti változásokról, szervezetről
- egyebekről

az egyedi adatok lapja ([pl.](https://kormany.hu/dokumentumtar/kozerdeku-adatok-megismeresere-vonatkozo-kormanyrendelet-tervezet)) tartalmazza:
- dokumentum neve
- publikálás dátuma
- minisztérium neve
- dokumentum típusa
- rövid leírás
- összes dokumentum (zip)
- dokumentumok egyenként (pdf)

**Comment by law909** [comment]
@pisipite érdemes forrás és/vagy dok.típusa szerint külön-külön figyuszt csinálni? A háttérben egy scraper lenne, de feliratkozni úgy lehetne, mintha több külön scraper lenne.
Nekem logikusnak tűnik forrás szerint külön figyusz.

**Comment by pisipite** [comment]
> @pisipite érdemes forrás és/vagy dok.típusa szerint külön-külön figyuszt csinálni? A háttérben egy scraper lenne, de feliratkozni úgy lehetne, mintha több külön scraper lenne. Nekem logikusnak tűnik forrás szerint külön figyusz.

igen, erre én is gondoltam. ha ügyesen meg lehet a frontenden oldani, hogy ne zsúfoljuk tele, akkor feltétlen. valószínű a forrás (minisztériumok) szerinti bontás lenne a legpraktikusabb, valami előbukkani radiobutton kiválasztóval.
@Klosmi mit gondolsz?

**Comment by Klosmi** [comment]
@pisipite szerintem lehetne egy accordionnal megoldani, ami kinyílik-bezárul, így el lehetne kerülni hogy zsúfolt legyen az oldal. Tehát az accordion lehetőséget nyújtana arra, hogy a felhasználó könnyen váltogathasson a különböző "minisztériumok" között, az egyes dokumentum típusok között.

**Comment by pisipite** [comment]
@boapps , ha nekifutsz ennek, lehessen szűkíteni a feliratkozást dokumentum típusa és forrása szerint is. mindkettő legyen kiválasztható (több kiválasztás engedélyezett).
frontendnek nyitottam [issue-t itt](https://github.com/Code-for-Hungary/bmm-frontend/issues/10).
